packages/types/src/feedback/index.ts

```diff
@@ -1,83 +1,2 @@
-import type { Attachment } from '../attachment';
-import type { Integration } from '../integration';
```

Comment:
Do we still need these (e.g. `types/src/feedback/attachment` etc) in types?

Reply:
I didn't look but assumed they were related to SendFeedback. I'll double check and either move/remove or dedupe as needed.

Reply:
turns out we didn't have a feedback-specific attachment type, it was a `..` import. 

but I still cleaned up the `feedback/` folder and put the leftover types into `types/src/feedback.ts`. It's got things that are part of core like `type SendFeedback` and `FeedbackEvent extends Event`